GetandTake/Program.cs

```diff
@@ -29,36 +42,33 @@
 builder.Services.RegisterServices(appSettings);
 builder.Services.RegisterDatabase(appSettings);
 
-
 var app = builder.Build();
 // Configure the HTTP request pipeline.
 if (!app.Environment.IsDevelopment())
 {
     app.UseExceptionHandler("/Error");
     // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
+    app.ExceptionHandler();
```

Comment:
It is recommended to place exception handler in general middleware pipeline, let's revert this change